src/main/java/org/medicmobile/webapp/mobile/AppUrlVerifier.java

```diff
@@ -87,3 +90,22 @@ public static AppUrlVerification failure(String appUrl, int failure) {
 		return new AppUrlVerification(appUrl, false, failure);
 	}
 }
+
+class AppUrlVerificationTask implements Callable<AppUrlVerification> {
+	private final String appUrl;
+
+	public AppUrlVerificationTask(String appUrl) {
+		this.appUrl = appUrl;
+	}
+
+	@Override
+	public AppUrlVerification call() {
+		trace(this, "AppUrlVerificationTask :: Executing call, appUrl=%s", appUrl);
+
+		if (Utils.isDebug() && (appUrl == null || appUrl.isEmpty())) {
```

Comment:
Not feeling strong about wrapping `BuildConfig.DEBUG` in `Utils.isDebug` for unit testing 🤷‍♀️

Reply:
MockStatic didn't work 🤔

Reply:
Wow! It appears that even the deep magic from `org.powermock.reflect.Whitebox.setInternalState` cannot touch the `BuildConfig` state....

Honestly, wrapping it feels like the right answer and your updates to the `Utils` class seem like the most straight-forward way to do this.  :+1: 

The only thing that bugs me is that mocking the static `Utils.isDebug` method is still annoying in the tests since you need the try-block and mockStatic stuff.  This is another great example of where a dependency injection framework would be helpful since we could just have an ordinary class (with non-static methods) that could act as our wrapper for `BuildConfig`. But instead of having to instantiate it everywhere or pass it into everything, the DI framework could just inject it for us.